src/organisms/feedPageNavBar/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface FeedPageNavBarProps {
+  label: string;
+  selectPlaceholder: string;
+  selectOptions: {};
+  selectValue: string | undefined;
+  selectOnChange: (e: any) => void;
+  account: string | null;
```

Comment:
make sure your types are consistent, in connect wallet button props account is optional param. 

I think overall its easier to maintain the types and avoid issues if we always inherit child props in the parent props.

What i mean by that is that if you need to pass  `ConnectWalletButton` prop into `FeePageNavBar` i would do:

```

export interface FeedPageNavBarProps {
    ...
} & Pick<ConnectWalletButtonProps, "account">

```

now if you would change the type of account inside the `ConnectWalletButtonProps` the feed will immediately be updated too.

@goshatravin let me know what you think

Reply:
Yes I agree @ArinCantCode , updated